src/config/config.ts

```diff
@@ -9,7 +9,9 @@ import { Locale } from '../i18n';
  *  For multi-level features, only the top-level featureName should be
  *  toggled.
  */
-export interface SandboxFeatures {}
+export interface SandboxFeatures {
+  Fintraffic: boolean;
+}
```

Comment:
The idea was to make a root Fintraffic folder, so that everything entirely company-specific stays there

Reply:
Yup, definitely! E.g. the the map in the Journey Patterns shall be as a standalone feature, but our custom tiles and default center/zoom level could go under /Fintraffic and then get supplied into the map providers